design/ui/pagination/cosmos.decorator.js

```diff
@@ -0,0 +1,11 @@
+// The only thing this decorator will do for now is import our SUI custom CSS.
+// There are currently issues with the out of the box compiled styles there
+// and possibly some missing loaders so we will use a different variant for now
+// that does not have any missing font/image dependencies.
+
+// import 'semantic-ui-css-custom';
```

Comment:
@lordpixel one the package you are working on is updated, using that custom CSS package here in this decorator should give us what we need. Note that the normalize CSS I THINK is included in that custom CSS. If not, we should do the normalize.css import above the custom SUI. If normalize is included in that custom SUI file then we can remove it here.

I think this should probably be included as a global cosmos decorator too for all instances of Cosmos being run. That way we do not have to define it again and again... and we can also see where/if there are any class collisions as we move forward to our own custom stuff.

Reply:
Again... no change needed. Just informational.